source/administration/config-settings.rst

```diff
@@ -1183,20 +1183,134 @@ Advanced Logging
 Output logs to multiple targets
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Allow any combination of local file, syslog, and TCP socket targets and send log records to multiple targets:
+Allow any combination of console, local file, syslog, and TCP socket targets, and send log records to multiple targets. These targets have been chosen to support the vast majority of log aggregators, and other log analysis tools, without having to install additional software.
 
-- Multiple local file targets: Supports rotation and compression triggered by size and/or duration.
-- Multiple syslogs: Supports local and remote syslog servers, with or without TLS transport.
-- Multiple TCP sockets: TCP socket target can be configured with an IP address or domain name, port, and optional TLS certificate.
+System Admins can define multiple log targets to:
 
-These three targets have been chosen to support the vast majority of log aggregators and other log analysis tools without having to install additional software. 
+- Mirror log output to files and log aggregators for redundancy.
+- Log certain entries to specific destinations. For example, all errors could be routed to a specific destination for review.
 
-Beyond the standard log levels (trace, debug, info, panic), discrete log levels can also be specified.
+Additional configuration options include:
 
-.. note::
-   Available discrete log levels are listed in ``mattermost-server:mlog/levels.go``. Logs are recorded asynchronously to reduce latency to the caller. Advanced logging supports hot-reloading of logger configuration.
+- Multiple local file targets: Supports rotation and compression triggered by size and/or duration.
+- Multiple syslogs: Supports local and remote syslog servers, with or without TLS transport.
+- Multiple TCP sockets: TCP socket target can be configured with an IP address or domain name, port, and optional TLS certificate.
 
-This feature's ``config.json`` setting is ``LogSettings.AdvancedLoggingConfig`` which can contain a filespec to another config file, a database DSN, or JSON. Options are outlined in this txt file: `Log Settings Options <https://github.com/mattermost/docs/files/5066579/Log.Settings.Options.txt>`_. Sample config: `Advanced Logging Options Sample.json.zip <https://github.com/mattermost/docs/files/5066597/Advanced.Logging.Options.Sample.json.zip>`_.
+All access to the REST API or CLI is audited. System Admins can configure whether the content of posts is included and whether permissions failures are audited.
```

Comment:
@wiggin77 - Can you tell me what config setting a System Admin would enable for this functionality, please? I'd like to include a link in the product documentation to the config setting.

Reply:
When using the AdvancedLogging config option for audit and specifying a file or JSON config, an admin can add one or more targets (log destinations) and decide what discrete log items to include in each target.  To capture everything they would use 
```
"Levels": [
    {"ID": 100, "Name": "audit-api"},
    {"ID": 101, "Name": "audit-content"},
    {"ID": 102, "Name": "audit-permissions"},
    {"ID": 103, "Name": "audit-cli"},
],
```
in the target config.   NOTE, I made a mistake in my previous answer.  `audit-content` does not include the content of posts.  That level is there to allow admins to turn off audit of every post/reaction as it can be a fire-hose for busy servers.  However many customers would need that enabled.

Reply:
`audit-api` - enables output of Rest API calls
`audit-content` - enables output of API calls that generate content (e.g. create post, create reaction)
`audit-permissions`- enables output of all permissions failures
`audit-cli` - enables output of legacy CLI calls

Reply:
Thanks @wiggin77! I've incorporated these details into this docs draft. Please review to ensure that I've captured the details accurately.